src/library_glfw.js

```diff
@@ -105,6 +107,7 @@ var LibraryGLFW = {
     versionString: null,
     initialTime: null,
     extensions: null,
+    devicePixelRatioMQS: null,
```

Comment:
What is MQS short for?

Reply:
It was supposed to be MediaQuerySearch but I realized that `window.matchMedia` returns a `MediaQueryList` not `MediaQuerySearch`... so I just renamed it `devicePixelRatioMQL` and added a comment for clarity